float64 min
int32 angle_min
float64 straight
int32 angle_straight
float64 right
int32 angle_right
float64 left
int32 angle_left

